### ISSUE TITLE
Requesting clarification

### ISSUE THREAD

**Issue by ivan-velikanova** [palexdev/MaterialFX#211]
In Reddit I read about MaterialFX a comment, which I supposed is by the core developers:

"Of course, every control in my library has its own new stylesheet. However, there are features that cannot be added with CSS. Take the buttons as an example. They are basically restyled with CSS but they also have a ripple effect on click and that feature cannot be implemented with CSS so the source code must be modified. That's also why I said that MaterialFX is not just a theme. A theme can be realized using only CSS but my library extends JavaFX controls and adds new features"

I see a similar ripple effect in "Gluon Mobile Light" which is presented as a theme. How are they doing it? 
Good work with material fx I really appreciate the work and I am tempted to use it, but I somehow want to stick with standard tools, features.

**Comment by palexdev** [comment]
Yes it was my comment

I don't know how they do it but... I think it's an easy guess. Even if it is presented as a theme I guess some of the controls for mobile have specific Skins to add features (like ripples) because in JavaFX there's no other way to do it

If I can give you some advice don't use just standard tools with JavaFX. Rather learn Maven or Gradle and start seeking for interesting third party framework. As I always say, JavaFX has a lot of potential and to really express it I really recommend to use libraries and frameworks to "complete" it.

See [AwesomeJavaFX](https://github.com/mhrimaz/AwesomeJavaFX) repo on GitHub for a curated list of all the guides, libraries and frameworks for JavaFX

**Comment by ivan-velikanova** [comment]
I have been using Maven for many years and even made a small tool to manage maven projects.
![image](https://user-images.githubusercontent.com/61929653/171028627-a099c61c-15ca-42a8-b01f-60e12976dbd6.png)

My fears with non-standard libraries/features are related to fragmentation, and things by some unfortunate circumstance ending up being unmaintained like JFoenix. I like your library, I appreciate your work, and I am amazed at the high-speed response. I think I will use your library (not that it is going to help you in anyway), but I really appreciate your prompt reply, and for that integrity that you show towards this project, I will surely pray for your success and prosperity and support your library (it is just a way I say thanks)

**Comment by palexdev** [comment]
Yeah I perfectly understand your concerns. But I also think that as long as a library/framework has all the things you need, doesn't have too many issues, it's good to go even if not supported anymore. ReactFX for example is a well know library by the JavaFX community (I personally dislike it) it's used a lot even if not supported anymore.
As for the fragmentation yeah, generally I tend to not import too many third-party libraries in my projects, just the ones I really need

Thanks for the support. To be honest I wish updates would be as quick as my responses haha. I should have released v11.14.0 a long ago but a lot is going on in my private life and in my projects. I'm currently focusing full-time on VirtualizedFX and it's a very very hard and time consuming task
